src/main/java/org/dsa/iot/knx/KnxConnection.java

```diff
@@ -54,7 +53,7 @@ public void remove() {
 		node.getParent().removeChild(node);
 	}
 
-	public abstract void onDiscovered();
+	public abstract void onDiscovered(boolean isRestore);
```

Comment:
This discovering function is used for different scenarios: discovering devices for command action or for restoring process.